exercises/list-ops/ListOps.cs

```diff
@@ -14,27 +14,27 @@ public static List<T> Reverse<T>(List<T> input)
         throw new NotImplementedException("You need to implement this function.");
     }
 
-    public static List<TOut> Map<TIn, TOut>(Func<TIn, TOut> map, List<TIn> input)
+    public static List<TOut> Map<TIn, TOut>(List<TIn> input, Func<TIn, TOut> map)
     {
         throw new NotImplementedException("You need to implement this function.");
     }
 
-    public static List<T> Filter<T>(Func<T, bool> predicate, List<T> input)
+    public static List<T> Filter<T>(List<T> input, Func<T, bool> predicate)
     {
         throw new NotImplementedException("You need to implement this function.");
     }
 
-    public static TOut Foldl<TIn, TOut>(Func<TOut, TIn, TOut> func, TOut start, List<TIn> input)
+    public static TOut Foldl<TIn, TOut>(List<TIn> input, TOut start, Func<TOut, TIn, TOut> func)
     {
         throw new NotImplementedException("You need to implement this function.");
     }
 
-    public static TOut Foldr<TIn, TOut>(Func<TIn, TOut, TOut> func, TOut start, List<TIn> input)
+    public static TOut Foldr<TIn, TOut>(List<TIn> input, TOut start, Func<TIn, TOut, TOut> func)
     {
         throw new NotImplementedException("You need to implement this function.");
     }
 
-    public static List<T> Concat<T>(List<List<T>> input)
+    public static List<object> Concat(List<object> input)
```

Comment:
Why change this to a `List<object>` as opposed to the more generic `List<T>`?

Reply:
Ah, that is a mistake. I'll correct it soon. Thanks for reviewing!

p.s. as you've noticed, I've done a lot of these generators the last couple of weeks. I requested a review on one of the first PR's, but having had no response for a couple of days, I figured you were either terribly busy or perhaps on holiday, which is why I've merged them myself. If you find anything not to your liking, please let me know!

p.p.s. Once all generators have been created, I'll be doing a refactoring of the generators code to hopefully simplify things.

Reply:
@robkeim Fixed!